.github/workflows/downstream.yaml

```diff
@@ -0,0 +1,50 @@
+name: IntegrationTest
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.package.repo }}/${{ matrix.package.group }}/${{ matrix.julia-version }}
+    runs-on: ${{ matrix.os }}
+    env:
+      GROUP: ${{ matrix.package.group }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version: [1,1.6]
+        os: [ubuntu-latest]
+        package:
+          - {user: JuliaPlots, repo: GraphMakie.jl, group: Downstream}
+          - {user: JuliaPlots, repo: AlgebraOfGraphics.jl, group: Downstream}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: x64
+      - uses: julia-actions/julia-buildpkg@latest
+      - name: Clone Downstream
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ matrix.package.user }}/${{ matrix.package.repo }}
+          path: downstream
+      - name: Load this and run the downstream tests
+        shell: julia --color=yes --project=downstream {0}
+        run: |
+          using Pkg
+          try
```

Comment:
```suggestion
           makie_version = TOML.parsefile("./Project.toml")["version"]
           downstream_project = TOML.parsefile("downstream/Project.toml")
           downstream_project["compat"]["Makie"] = makie_version

           write("downstream/Project.toml") do io
               TOML.print(io, downstream_project)
           end

           try
```

Reply:
We could try something like this to actually edit the downstream's Project.toml to be compatible with Makie's current master version.

Reply:
thanks, this is still really useful